http-server-netty/src/test/groovy/io/micronaut/http/server/netty/websocket/BinaryChatClientWebSocket.java

```diff
@@ -70,9 +70,8 @@ public WebSocketSession getSession() {
     }
 
     @OnMessage
-    public void onMessage(
-            byte[] message) {
-        System.out.println("Client received message = " + new String(message));
+    public void onMessage(byte[] message) {
+        System.out.println("Client for " + username + " on topic " + topic + " received message = " + new String(message));
```

Comment:
delete print statement?

Reply:
These are there to try and see what is going on...  Once we work this out, they can and will be removed